src/main.ts

```diff
@@ -1,26 +1,43 @@
+import { getActionInputs, getOrganisationalUnits } from './helpers'
+import { ActionInput, WorkloadAccount } from './types'
+import { loadAccounts, writeAccounts } from './parser'
 import * as core from '@actions/core'
-import { wait } from './wait'
 
 /**
  * The main function for the action.
  * @returns {Promise<void>} Resolves when the action is complete.
  */
 export async function run(): Promise<void> {
   try {
-    const ms: string = core.getInput('milliseconds')
+    // Get the GH token and version file path
+    const inputs: ActionInput = getActionInputs([
+      { name: 'file_path', options: { required: true } },
+      { name: 'customer_id', options: { required: true } },
+      { name: 'spoc_email', options: { required: true } },
```

Comment:
We might want to add validation around this to ensure its an email? Will we have valid domains we would only accept?

Reply:
The more I think about this the more I'm starting to think we don't want true indepth email checking. We might want a basic regex check + length check as the local part cannot exceed 64 characters as per: https://www.rfc-editor.org/rfc/rfc5321#section-4.5.3.1.1

Reply:
Yep, I don't think we want to actually validate the email address, more so ensure that the format matches that of a correct email. Might also want to ensure uniqueness of email address.

Reply:
Initial change has ensured uniqueness of email address from existing records.